test/system/255-auto-update.bats

```diff
@@ -261,7 +261,8 @@ EOF
     systemctl enable --now podman-auto-update-$cname.timer
     systemctl list-timers --all
 
-    local expect='Finished Podman auto-update testing service'
+    # While newer systemd uses 'Finished', older systemd uses 'Started' for oneshot services
```

Comment:
A good habit to develop: never use "new", "recent", "old" in a code comment. A future maintainer, either tomorrow or in five years, will have no possible way of knowing what that means. Could you track down a systemd version tag and use that instead?